doc/quickstart_in_rviz/quickstart_in_rviz_tutorial.rst

```diff
@@ -131,10 +131,13 @@ Note what happens when you try to move an end-effector out of its reachable work
 
 Moving Joints or in Null Space
 ++++++++++++++++++++++++++++++
-You can use the **Joints** tab to move single joints and the redundant joints of 7-DOF robots. Try moving the "null space exploration" slider as shown in the animation below.
+You can use the **Joints** tab to move single joints and the redundant joints of 7-DOF robots. Try moving the "null space exploration" slider as shown in this animation.
 
-.. image:: rviz_joints_nullspace.webm
-   :width: 700px
+.. raw:: html
+
+    <div style="position: relative; padding-bottom: 5%; height: 0; overflow: hidden; max-width: 100%; height: auto;">
+        <iframe width="700px" height="400px" src="../../../../doc/quickstart_in_rviz/rviz_joints_nullspace.webm" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
```

Comment:
+1 to @rhaschke's comment. A relative path seems brittle. There has to be a better way to do this

Reply:
@mlautman: Why do you then approve?
The relative path here is transformed into an absolute path by sphinx, which will be broken then on the web.

Reply:
I think the way to go is to use the [video plugin for sphinx](https://github.com/sphinx-contrib/video) that @felixvd mentioned.
@JafarAbdi, can you have a look at this, please?

Reply:
@rhaschke I approved because I didn't have a better suggestion and it seemed to fix the problem. Also, there are a lot of other relative paths in the tutorials, albeit none as aggressive as this, so I didn't think that my reservations were disqualifying.

Reply:
While other relative paths stay relative, this one here becomes absolute and will - for this reason - break. Ping @JafarAbdi to have a look at the proposed video plugin.

Reply:
@rhaschke Sorry for the delay, I just saw your messages -- another option could be replacing

```html
<iframe width="700px" height="400px" src="../../../../doc/quickstart_in_rviz/rviz_joints_nullspace.webm" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
```

with

```html
<video width="700px" height="400px" controls src="../../../../doc/quickstart_in_rviz/rviz_joints_nullspace.webm"></video>
```

which will keep the URL relative rather than absolute

Reply:
I'm fine with any working approach.

Reply:
I tried to build the doc on my fork, [but Github pages gives an error](https://user-images.githubusercontent.com/4535737/92940049-92465080-f489-11ea-9d01-8aee83766a18.png). I suppose there's more to it. Is there another way to check that this works, or fix this @JafarAbdi ?

Reply:
Running the following line should be sufficient https://github.com/ros-planning/moveit_tutorials/blob/master/.travis.yml#L47-L52

Reply:
Sorry for the delay, I remember having trouble setting up the website locally but I don't how I got it working later, this [PR](https://github.com/ros-planning/moveit_tutorials/pull/546) fix the issue by converting it to GIF

Reply:
I commented over there. Figuring out how to embed the webm would really be better than encoding video in a 30+ year old, barely compressed format.